empress/clumpr/Histogram.py

```diff
@@ -168,73 +168,3 @@ def __mul__(self, other):
     def __repr__(self):
         return str(self.histogram_dict)
 
- # Tests
-
-if __name__ == '__main__':
-
-    def testShift1():
-        hist = Histogram(None)
-        new_hist = hist << 1
-
-        assert(new_hist.histogram_dict == {})
-    
-    def testShift2():
-        hist = Histogram(0)
-        new_hist = hist << 3
-
-        assert(hist.histogram_dict == {0:1})
-        assert(new_hist.histogram_dict == {3:1})
-    
-    def testShift3():
-        hist = Histogram({0:1, 3:5})
-        new_hist = hist << 1
-
-        assert(hist.histogram_dict == {0:1, 3:5})
-        assert(new_hist.histogram_dict == {1:1, 4:5})
-    
-    def testCombine1():
-        histA = Histogram(0)
-        histB = Histogram(None)
-        new_hist = histA + histB
-
-        assert(histA.histogram_dict == {0:1})
-        assert(histB.histogram_dict == {})
-        assert(new_hist.histogram_dict == {0:1})
-    
-    def testCombine2():
-        histA = Histogram({0:1, 2:1, 3:2})
-        histB = Histogram({2:1, 3:1, 10:1})
-        new_hist = histA + histB
-
-        assert(histA.histogram_dict == {0:1, 2:1, 3:2})
-        assert(histB.histogram_dict == {2:1, 3:1, 10:1})
-        assert(new_hist.histogram_dict == {0:1, 2:2, 3:3, 10:1})
-
-    def testProduct1():
-        # TODO: update test to reflect new product function
-        histA = Histogram(0)
-        histB = Histogram(None)
-        new_hist = histA * histB
-
-        assert(histA.histogram_dict == {0:1})
-        assert(histB.histogram_dict == {})
-        assert(new_hist.histogram_dict == {})
-    
-    def testProduct2():
-        # TODO: update test to reflect new product function
-        histA = Histogram({0:1, 2:1, 3:2})
-        histB = Histogram({2:1, 3:1, 5:1})
-        new_hist = histA * histB
-
-        assert(histA.histogram_dict == {0:1, 2:1, 3:2})
-        assert(histB.histogram_dict == {2:1, 3:1, 5:1})
-        assert(new_hist.histogram_dict == {2:1, 3:1, 4:1, 5:4, 6:2, 7:1, 8:2})
-
-    testShift1()
-    testShift2()
-    testShift3()
-    testCombine1()
-    testCombine2()
-    testProduct1()
-    testProduct2()
```

Comment:
Are you positive about deleting this instead of moving it elsewhere?

Reply:
Could you create a unit test file for this in the test folder?

Reply:
Let me try if I could

Reply:
I managed and tested with
running `python -m unittest discover "./test/clumpr"`